docs/encyclopedia/temporal-nexus-service-operations.mdx

```diff
@@ -0,0 +1,146 @@
+   5. Operation `id` is added to the caller’s operation `OperationHandle`
+4. Handler’s Nexus Machinery sends a Nexus completion `Callback` when the `Workflow` has completed.
+   1. Operation `result` is returned via callback since the operation is asynchronous
+   2. **`NexusOperationCompleted`** added to caller’s event history
+5. Caller’s Workflow `handle.GetResult()` returns the result
+
+##### Synchronous Operations Lifecycle
+
+An synchronous Nexus Operation may take up to 10 seconds to complete and will then timeout.
+
+The lifecycle of a synchronous Nexus `Operation`, for example to do a `Query` or a `Signal`:
+
+1. Caller’s Workflow does a `myServiceClient.ExecuteOperation()`
+   1. **`NexusOperationScheduled`** added to caller’s event history
+   2. Nexus `Operation` added to Nexus Machinery in the caller’s Temporal cluster
+   3. Immediately returns an `OperationHandle`, so the Workflow does not block.
+   4. Caller’s Workflow can interact with the `Operation` using the `OperationHandle`
+      1. `WaitStarted()`, `GetResult()`
+2. Start Nexus operation on the target cluster is attempted by the caller’s Temporal cluster
+   1. Via the Nexus Machinery in the target cluster
+   2. Retries are processed by the Nexus Machinery in the caller’s Temporal cluster and tracked as part of the caller Workflow, similar to Activities.
+3. Nexus `Operation handler` in the target cluster processes the Nexus `Operation`
+   1. `Operation handler` pulls operation `Task` from Task Queue
+   2. `Operation handler` starts a `Workflow` in the target cluster
+   3. Operation `result` is returned inline, since the operation is synchronous
+   4. **`NexusOperationCompleted`** added to caller’s event history
+4. Caller’s Workflow `handle.GetResult()` returns the result
+
+##### Executing Arbitrary (non-durable code) from a Sync Handler {#executing-arbitrary-(non-durable-code)-from-a-sync-handler}
+
+Nexus `Operation handlers` can execute arbitrary code, but unlike Activities they should be relatively short-lived, due to the lack of [heartbeats](https://community.temporal.io/t/best-practices-for-long-running-Activities/934) and a default 10 second NexusOperationStarted timeout. Nexus `Operation handlers` are not required to use underlying Temporal primitives like a `Workflow` or `Signal`.
+
+#### Automatic Retries
+
+Once the caller `Workflow` schedules an `Operation` with the caller’s Temporal cluster, the caller’s Nexus Machinery keeps trying to start the `Operation`, with automatic retries and exponential backoff. If a Nexus `Operation` returns a [retryable error](https://github.com/temporalio/temporal/blob/13d6cd8cf7a4ba0c4660cf98f672bbd645dca3e7/components/nexusoperations/executors.go#L659) when attempting to start, the `Operation` it will be retried up to the [default retry policy’s](https://github.com/temporalio/temporal/blob/de7c8879e103be666a7b067cc1b247f0ac63c25c/components/nexusoperations/config.go#L111) max attempts and expiration interval.
```

Comment:
```suggestion
Once the caller `Workflow` schedules an `Operation` with the caller’s Temporal cluster, the caller’s Nexus Machinery keeps trying to start the `Operation`, with automatic retries and exponential backoff. 
If a Nexus `Operation` returns a [retryable error](https://github.com/temporalio/temporal/blob/13d6cd8cf7a4ba0c4660cf98f672bbd645dca3e7/components/nexusoperations/executors.go#L659) when attempting to start, the `Operation` it will be retried up to the [default retry policy’s](https://github.com/temporalio/temporal/blob/de7c8879e103be666a7b067cc1b247f0ac63c25c/components/nexusoperations/config.go#L111) max attempts and expiration interval.
```